DEVELOPMENT.md

```diff
@@ -19,47 +92,40 @@ To check out this repository:
 1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of this repo
 2. Create the directories and clone your fork
 
-```
-mkdir -p $GOPATH/src/github.com/KohlsTechnology
-cd $GOPATH/src/github.com/KohlsTechnology
-git clone https://github.com/<YOUR FORK>/eunomia.git
+```shell
+git clone https://github.com/<YOUR_FORK>/eunomia.git
 cd eunomia
 ```
 
-### Installing on a Mac
+## Running Locally for Development Purposes
+
+The most efficient way to develop the operator locally is run the code on your local machine. This allows you to test code changes as you make them.
 
-All the components can easily be installed via [Homebrew](https://brew.sh/) on a Mac:
+To achieve this, run the helper script (preferably in a separate console window as the script blocks):
 
 ```shell
-brew install git
-brew install go
-brew install docker
-brew install operator-sdk
-brew install minikube
-brew install minishift
+./scripts/run-eunomia-locally.sh
 ```
 
-## Running Locally for Development Purposes
+If you want Eunomia to watch a particular namespace, just pass its name as an argument to the above script. Running the script without the argument defaults to Eunomia watching all namespaces.
 
-The most efficient way to develop the operator locally is run the code on your local machine. This allows you to test code changes as you make them.
+### **Note on cluster being watched**
 
-```
-minikube start
-kubectl apply -f ./deploy/crds/eunomia_v1alpha1_gitopsconfig_crd.yaml
-export JOB_TEMPLATE=./build/job-templates/job.yaml
-export CRONJOB_TEMPLATE=./build/job-templates/cronjob.yaml
-export WATCH_NAMESPACE=""
-export OPERATOR_NAME=eunomia-operator
-export GO111MODULE=on
-go mod vendor
-operator-sdk up local --namespace="${WATCH_NAMESPACE}"
+The above script uses `operator-sdk up local` command to run Eunomia operator binary locally. **It enforces the use of a Kubernetes cluster** - Eunomia will be watching namespace(s) in a Kubernetes cluster. Using an OpenShift cluster is unconfigurable, unfortunately.
```

Comment:
[optional] I don't really understand what the "unconfigurable" sentence means; what would you think about removing it?

```suggestion
The above script uses `operator-sdk up local` command to run Eunomia operator binary locally. **It enforces the use of a Kubernetes cluster** - Eunomia will be watching namespace(s) in a Kubernetes cluster.
```